### ISSUE TITLE
set background to Colors.transparent ，But it didn't take effect

### ISSUE THREAD

**Issue by anjingxz** [afonsocraposo/buttons_tabbar#51]
## Description
set background to Colors.transparent ，But it didn't take effect

<img width="336" alt="image" src="https://github.com/afonsocraposo/buttons_tabbar/assets/1071010/e12912b9-a6b8-4b77-9e11-d339ac97a442">

## Example

```dart
ButtonsTabBar(
  duration: 30,
  height: 24,
  buttonMargin: const EdgeInsets.fromLTRB(16, 0, 16, 0),
  contentPadding: const EdgeInsets.symmetric(
    horizontal: 8.0,
    vertical: 2.0,
  ),
  backgroundColor: ColorUtil("#40C8E0"),
  unselectedBackgroundColor: Colors.transparent,
  decoration: null,
  unselectedDecoration: null,
  radius: 5.0,
  tabs: const [
    Tab(text: "Pomodoro"),
    Tab(text: "Short Break"),
    Tab(text: "Long Break"),
  ],
)
```

Thank you for your feature request! We value your feedback and appreciate your contribution to making our product better.

**Comment by Amdian** [comment]
I encountered the same problem, and now I changed the version to 1.3.8. I really hope that when you update the version, you can test it yourself, because this will greatly affect the people who use it.

**Comment by afonsocraposo** [comment]
Let me check it 🫡

**Comment by afonsocraposo** [comment]
@anjingxz and @Amdian thank you for raising this issue!
It's fixed in version `1.3.10`:
https://pub.dev/packages/buttons_tabbar/versions/1.3.10
